aspnet/index.rst

```diff
@@ -5,10 +5,10 @@
 
 .. _index:
 
-ASP.NET 5 Documentation
+ASP.NET Core 1.0 Documentation
 =======================
```

Comment:
The underline need to be the full length of the [title or section header](http://www.sphinx-doc.org/en/stable/rest.html#sections).